src/node_report_utils.cc

```diff
@@ -141,16 +135,14 @@ void WalkHandle(uv_handle_t* h, void* arg) {
       }
       break;
     }
-    case UV_SIGNAL: {
+    case UV_SIGNAL:
       // SIGWINCH is used by libuv so always appears.
       // See http://docs.libuv.org/en/v1.x/signal.html
-      data << "signum: " << handle->signal.signum
+      data << "signum: " << handle->signal.signum;
 #ifndef _WIN32
-           << " (" << node::signo_string(handle->signal.signum) << ")"
+      data << " (" << node::signo_string(handle->signal.signum) << ")";
```

Comment:
This is wrapped in `#ifndef _WIN32` because the standalone module wasn't able to access the function on Windows since it's not exported (https://github.com/nodejs/node-report/issues/100). In core the guard can probably be removed.